### DISCUSSION TITLE
Support For Row Level Security

### DISCUSSION THREAD

**Discussion by magicMustard** [aarondl/sqlboiler#943] [Q&A] [answered]
Hello,

I'm looking into your project and looks very interesting. However, I have a question about support for Row Level Security.

Does your project support Row Level Security?

**Comment by aarondl** [comment, marked as answer]
Hi there @magicMustard. Apologies for the late response. Life's busy. There is no security support at all in the project at this time (and probably won't ever be).

**Reply by magicMustard** [reply]
No worries, thank you
